src/script/api/script_road.hpp

```diff
@@ -85,6 +85,42 @@ class ScriptRoad : public ScriptObject {
 		BT_TRUCK_STOP, ///< Build a truck stop
 	};
 
+	/**
+	 * A bitmap of all the possible road tiles and combinations.
+	 */
+	enum RoadTiles {
```

Comment:
This isn't about tiles at all. It's probably be better to rename it something like RoadBits or RoadPieces or similar?